zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGeneratorTest.scala

```diff
@@ -530,6 +530,70 @@ class AQLGeneratorTest extends Specification {
           |}""".stripMargin.trim)
     }
 
+    "translate lookup one table with one join key" in {
+      val populationDataSet = PopulationDataStore.DatasetName
+      val populationSchema = PopulationDataStore.PopulationSchema
+
+      val selectValues = Seq("*", "population")
+      val selectStatement = SelectStatement(Seq.empty, 0, 0, selectValues)
+      val lookup = LookupStatement(Seq("geo_tag.stateID"), populationDataSet, Seq("stateID"), selectValues,
+        selectValues)
+      val filter = Seq(textFilter)
+      val query = new Query(TwitterDataSet, Seq(lookup), filter, Seq.empty, select = Some(selectStatement))
+      val result = parser.generate(query, Map(TwitterDataSet -> schema, populationDataSet -> populationSchema))
+      removeEmptyLine(result) must_== unifyNewLine(
+        """
+          |for $t in dataset twitter.ds_tweet
+          |for $l0 in dataset twitter.US_population
```

Comment:
wait, I think we are in the wrong direction. Here we want a *left-out-join* not an equal join.
I found one using SQL++ [here](https://ci.apache.org/projects/asterixdb/sqlpp/manual.html#Left_outer_joins), @waans11 do you know what the corresponding AQL syntax for that ?